.circleci/config.yml

```diff
@@ -93,8 +93,11 @@ jobs:
       - restore_cache:
           key: dependency-cache-{{ checksum "package-lock.json" }}
       - run:
-          name: Get wiremock # match version in docker-compose-test.yml
-          command: curl -o wiremock.jar https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-jre8-standalone/2.35.0/wiremock-jre8-standalone-2.35.0.jar
+          name: Get wiremock (latest version)
```

Comment:
See here for details: https://github.com/ministryofjustice/hmpps-book-a-prison-visit-ui/pull/21#discussion_r1334317567